sycl/test/lit.cfg.py

```diff
@@ -150,6 +150,8 @@ def getDeviceCount(device_type):
     config.available_features.add('gpu')
     if cuda:
        config.available_features.add('cuda')
+       gpu_run_substitute += " SYCL_BE=PI_CUDA "
```

Comment:
Adapt/rebase on top of https://github.com/intel/llvm/pull/1288 instead which has a more elegant and simpler solution.

Reply:
I decided to not adapt the mentioned [PR](https://github.com/intel/llvm/pull/1288) as setting the environment is not visible in LIT's output when a command fails and therefore makes it harder to copy and paste command invocation to then run and inspect by hand.